pkg/sql/control_schedules.go

```diff
@@ -132,6 +137,16 @@ func (n *controlSchedulesNode) startExec(params runParams) error {
 			return err
 		}
 
+		isAdmin, err := params.p.UserHasAdminRole(params.ctx, params.p.User())
+		if err != nil {
+			return err
+		}
+		isOwner := schedule.Owner() == params.p.User()
+		if !isAdmin && !isOwner {
+			return pgerror.Newf(pgcode.InsufficientPrivilege, "must be admin or owner of the "+
+				"schedule %d to %s it", schedule.ScheduleID(), n.command.String())
```

Comment:
is `n.command.String()` helpful info here?

Reply:
🤷 it makes for a more informative error message that you're not allowed to `RESUME`, `DROP`, `PAUSE` it, but I guess you already know that because you're running the command.